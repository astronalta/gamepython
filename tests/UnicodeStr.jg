# jgtest exec on


main() Int {
    str = "По оживлённым берегам
Громады стройные теснятся
Дворцов и башен; корабли
Толпой со всех концов земли
К богатым пристаням стремятся;
"
    Io::print(str)
    Io::println("По оживлённым берегам".chars.str)
    Io::println("Громады стройные теснятся".chars.str)
    Io::println("Дворцов и башен; корабли".chars.str)
    Io::println("Толпой со всех концов земли".chars.str)
    Io::println("К богатым пристаням стремятся;".chars.str)
    Io::println(str.chars.str)
    ret 0
}

# String::slice
# String::prefix
# String::suffix
# String::@index
# String::toupper
# String::tolower
# Char::str?
