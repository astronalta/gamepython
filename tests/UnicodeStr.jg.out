По оживлённым берегам
Громады стройные теснятся
Дворцов и башен; корабли
Толпой со всех концов земли
К богатым пристаням стремятся;
21
25
24
27
30
132
